build/libllvm.ps1

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
```suggestion
# Copyright (c) Microsoft Corporation.
```

Reply:
@cgranade This is the format used in 13 powershell files, 7 C#, 91 F# files

Reply:
Yes, and policy used to require it, but that policy was updated ~two years ago, and we are now asked to not include the phrase "all rights reserved" in new code. A lot of older code still uses the old policy, and sometimes the phrase slips through code reviews.